lib/basedriver/commands/session.js

```diff
@@ -17,21 +17,26 @@ commands.createSession = async function (jsonwpDesiredCapabilities, jsonwpRequir
   // Determine weather we should use jsonwpDesiredCapabilities or w3cCapabilities to get caps from
   let caps;
   if (w3cCapabilities) {
+    this.setProtocolW3C();
```

Comment:
I guess I would have expected these to be like `this.setProtocol(BaseDriver.DRIVER_PROTOCOL.W3C)` but it's all the same